examples/react-todo/tailwind.config.cjs

```diff
@@ -11,12 +11,6 @@ module.exports = {
   content: ["./**/*.{html,tsx}"],
   theme: {
     extend: {
-      fontFamily: {
-        mono: [mono, emoji, "monospace"],
-        sans: [sans, emoji, "sans-serif"],
-        condensed: [condensed, emoji, "sans-serif"],
-        serif: [serif, emoji, "serif"],
-      },
```

Comment:
It seems fine to rely on [the defaults](https://tailwindcss.com/docs/font-family).